hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/debug/TestFindContainerKeys.java

```diff
@@ -0,0 +1,286 @@
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.apache.hadoop.hdds.protocol.proto.HddsProtos.ReplicationFactor.ONE;
+import static org.apache.hadoop.ozone.OzoneConsts.OM_KEY_PREFIX;
+import static org.apache.hadoop.ozone.om.OmMetadataManagerImpl.DIRECTORY_TABLE;
+import static org.apache.hadoop.ozone.om.OmMetadataManagerImpl.FILE_TABLE;
+import static org.apache.hadoop.ozone.om.OmMetadataManagerImpl.KEY_TABLE;
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Test class for {@link FindContainerKeys}.
+ */
+public class TestFindContainerKeys {
+  private DBStore dbStore;
+  @TempDir
+  private File tempDir;
+  private StringWriter stdout, stderr;
+  private PrintWriter pstdout, pstderr;
+  private CommandLine cmd;
+  private static final Gson GSON = new GsonBuilder().setPrettyPrinting().create();
+  private String[] cmdArgs;
+
+  @BeforeEach
+  public void setup() throws IOException {
+    OzoneConfiguration conf = new OzoneConfiguration();
+    stdout = new StringWriter();
+    pstdout = new PrintWriter(stdout);
+    stderr = new StringWriter();
+    pstderr = new PrintWriter(stderr);
+
+    cmd = new CommandLine(new OzoneDebug())
+        .addSubcommand(new ContainerCommands())
+        .addSubcommand(new FindContainerKeys())
+        .setOut(pstdout)
+        .setErr(pstderr);
+
+    dbStore = DBStoreBuilder.newBuilder(conf).setName("om.db")
+        .setPath(tempDir.toPath()).addTable(KEY_TABLE).addTable(FILE_TABLE)
+        .addTable(DIRECTORY_TABLE)
+        .build();
+
+    cmdArgs =
+        new String[]{"find-keys", "--om-db", dbStore.getDbLocation().getAbsolutePath(), "--container-ids", "1 2 3"};
+  }
+
+  @AfterEach
+  public void shutdown() throws IOException {
+    closeDbStore();
+    pstderr.close();
+    stderr.close();
+    pstdout.close();
+    stdout.close();
+  }
+
+  @Test
+  void testFSO() throws Exception {
+    /*
+      Structure:
+      keyName (container id)
+
+      /vol1/bucet1
+        - key1 (1)
+        - dir1
+          - key2 (2)
+          - dir2
+            - key3 (3)
+            - key4 (3)
+        - key5 (4)
+     */
+    long volumeId = -123L;
+    long bucketId = -456L;
+    long dirObjectId1 = -789L;
+    long dirObjectId2 = -788L;
+    createDirectory(volumeId, bucketId, bucketId, dirObjectId1, "dir1");
+    createDirectory(volumeId, bucketId, dirObjectId1, dirObjectId2, "dir2");
+    createFile(volumeId, bucketId, "key1", -987L, bucketId, 1L);
+    createFile(volumeId, bucketId, "key2", -986L, dirObjectId1, 2L);
+    createFile(volumeId, bucketId, "key3", -985L, dirObjectId2, 3L);
+    createFile(volumeId, bucketId, "key4", -984L, dirObjectId2, 3L);
+    createFile(volumeId, bucketId, "key5", -983L, dirObjectId2, 4L);
+
+    closeDbStore();
```

Comment:
Why do we need to close it in individual test when we are closing it in `@AfterEach`?